starter-code/index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Home - revera</title>
+  <link rel="stylesheet" type="text/css" href="css/main.css">
```

Comment:
Your custom css file should be linked AFTER any library css like bootstrap. That way, the bootstrap css won't override your own.